tensorboard/webapp/webapp_data_source/tb_feature_flag_data_source.ts

```diff
@@ -98,6 +97,40 @@ export class QueryParamsFeatureFlagDataSource
     return featureFlags;
   }
 
+  persistFeatureFlags(flags: Partial<FeatureFlags>) {
+    const currentState = this.getPersistentFeatureFlags();
+    const newState = {
+      ...currentState,
+      ...flags,
+    };
+    localStorage.setItem(FEATURE_FLAG_STORAGE_KEY, JSON.stringify(newState));
+  }
+
+  resetPersistentFeatureFlag<K extends keyof FeatureFlags>(featureFlag: K) {
+    const currentState = this.getPersistentFeatureFlags();
+    if (currentState[featureFlag] == undefined) {
+      return;
+    }
+    delete currentState[featureFlag];
+    if (Object.keys(currentState).length === 0) {
+      localStorage.removeItem(FEATURE_FLAG_STORAGE_KEY);
```

Comment:
Might be worth adding a brief comment what is happening here.

Reply:
Added.